public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -4,10 +4,14 @@
  * Plugin Description: Sessions, Speakers, Sponsors and much more.
  */
 
-require( 'inc/back-compat.php' );
-require_once( 'inc/favorite-schedule-shortcode.php' );
-require_once( 'inc/privacy.php' );
-require_once( 'inc/deprecated.php' );
+// Disable phpcs rules for strict comparisons, since fixing these might break edge cases.
+// phpcs:disable WordPress.PHP.StrictComparisons.LooseComparison
+// phpcs:disable WordPress.PHP.StrictInArray.MissingTrueStrict
```

Comment:
`SHOULD` - I don't think we should ignore CS violations simply because they require work/testing to fix. I'd rather leave them there until we have time to properly fix them, or just fix them with minimal testing, and then come back and fix it if that introduces a bug that's discovered later on.

Reply:
I disagree, I think leaving the failing tests in leads to everyone ignoring failing tests. We have PHPCS running on all PRs, and IMO we should not merge anything [with failing checks.](https://github.com/WordPress/wordcamp.org/pull/220/checks)

I'm happy to fix these now, in this PR, but I didn't want to get flagged for unrelated changes.

> I'd rather leave them there until we have time to properly fix them

That's essentially what this does– now we can come back, search for `phpcs:disable`, remove the disable rule and fix the issues.

Reply:
>  I think leaving the failing tests in leads to everyone ignoring failing tests

I agree with that.

> IMO we should not merge anything with failing checks.

If the failure is just because of CS stuff, merging it wouldn't bother me personally.

> I didn't want to get flagged for unrelated changes

I'm fine w/ CS fixes in a PR, as long as they're a separate commit.

> That's essentially what this does– now we can come back, search for phpcs:disable, remove the disable rule and fix the issues.

I don't think that _fixes_ the issue, it just _hides_ it. I think a better solution would be to [make the linters only run on changed lines](https://make.wordpress.org/core/handbook/best-practices/commit-messages/#before-a-commit), or fix the CS issues in a separate commit, or merge the PR despite the CS failures, or make CS failures a warning instead of a failure condition (if that's possible).

Reply:
I've removed the comments– these are just warnings, so we'll let it through.